sensor_msgs/msg/BatteryState.msg

```diff
@@ -41,14 +41,23 @@ float32 charge           # Current charge in Ah  (If unmeasured NaN)
 float32 capacity         # Capacity in Ah (last full capacity)  (If unmeasured NaN)
 float32 design_capacity  # Capacity in Ah (design capacity)  (If unmeasured NaN)
 float32 percentage       # Charge percentage on 0 to 1 range  (If unmeasured NaN)
+uint32  id               # Identifier for the battery
 uint8   power_supply_status     # The charging status as reported. Values defined above
 uint8   power_supply_health     # The battery health metric. Values defined above
 uint8   power_supply_technology # The battery chemistry. Values defined above
 bool    present          # True if the battery is present
 
+uint32  charge_fault     # The current charge fault of the battery, if there is one
```

Comment:
These seem to be in a potential alternative value to the power_supply_health instead of a new field. 

Related for these new fields the proposed ones don't have clear interpretation and wouldn't be acceptable as defined.